lib/kafka/datadog.rb

```diff
@@ -40,7 +40,7 @@ def statsd=(statsd)
       end
 
       def host
-        @host ||= ::Datadog::Statsd::DEFAULT_HOST
+        @host ||= ::Kafka::Statsd::DEFAULT_HOST
```

Comment:
This change wasn't part of #654 and will break compatibility with dogstatsd-ruby < 4.0. See https://github.com/zendesk/ruby-kafka/pull/634/files.

Reply:
I see. So, I'll cherry pick the changes in that PRs to backport to 0.6.x too.